ngraph/core/reference/src/runtime/reference/tile.cpp

```diff
@@ -14,34 +14,93 @@
 // limitations under the License.
 //*****************************************************************************
 
+#include <algorithm>
 #include <cmath>
-#include <stdio.h>
+#include <cstdio>
+#include <numeric>
 
 #include "ngraph/check.hpp"
 #include "ngraph/runtime/reference/tile.hpp"
 
 using namespace ngraph;
 
-void runtime::reference::tile(
-    const char* arg, char* out, const Shape& in_shape, const Shape& out_shape, size_t elem_size)
+namespace
+{
+    bool Increase(const Shape& shape, std::vector<int64_t>& indices, size_t& axis, bool& run)
```

Comment:
nGraph's naming convention is snake_case, not CamelCase. 
```suggestion
    bool increase(const Shape& shape, std::vector<int64_t>& indices, size_t& axis, bool& run)
```
Also, this name is not very informative. What is increasing?